client/app/src/main/java/com/microsoft/research/karya/data/service/MicroTaskAPI.kt

```diff
@@ -1,32 +1,26 @@
 package com.microsoft.research.karya.data.service
 
+import com.google.gson.JsonArray
 import com.microsoft.research.karya.data.model.karya.KaryaFileRecord
+import com.microsoft.research.karya.data.remote.response.GetAssignmentsResponse
 import okhttp3.MultipartBody
 import okhttp3.ResponseBody
+import org.json.JSONArray
 import retrofit2.Response
-import retrofit2.http.GET
-import retrofit2.http.Header
-import retrofit2.http.Multipart
-import retrofit2.http.POST
-import retrofit2.http.Part
-import retrofit2.http.Path
+import retrofit2.http.*
 
 interface MicroTaskAPI {
 
-    @Multipart
-    @POST("/microtask_assignment/{id}/output_file")
-    suspend fun postUploads(
-        @Header("auth-provider") authProvider: String,
+    @PUT("/assignments")
+    suspend fun submitAssignments(
         @Header("id-token") idTokenHeader: String,
-        @Path("id") microtaskAssignmentID: String,
-        @Part json: MultipartBody.Part,
-        @Part file: MultipartBody.Part
-    ): Response<KaryaFileRecord>
+        @Body updates: JsonArray
```

Comment:
@anuragshukla06 sorry I missed this one, can you also update this to take a specific type instead of a `JsonArray`?

Reply:
No No, I am sorry I missed this. Done